guac/certifier-scorecard.md

```diff
@@ -0,0 +1,233 @@
+- **Performance**: Slower, resource-intensive
+- **Coverage**: Any accessible GitHub repository
+
+## Available Options
+
+### Usage
+
+Basic command syntax:
+
+```bash
+guaccollect scorecard [options]
+```
+
+### Core Flags
+
+| Flag                         | Description                                                    | Default             |
+| ---------------------------- | -------------------------------------------------------------- | ------------------- |
+| `--certifier-batch-size int` | Sets the batch size for pagination query for the certifier     | 60000               |
+| `--certifier-latency string` | Sets artificial latency on the certifier (e.g., m, h, s, etc.) | Not enabled (empty) |
+| `-h, --help`                 | Help for scorecard                                             |                     |
+| `--interval string`          | Polling interval (e.g., m, h, s, etc.)                         | 5m                  |
+| `--service-poll`             | Enable polling mode                                            | false               |
+
+### Scorecard-Specific Flags
+
+| Flag                        | Description                                       | Default                              |
+| --------------------------- | ------------------------------------------------- | ------------------------------------ |
+| `--scorecard-fetcher-type`  | Fetcher type: `api` (default) or `local`          | `api`                                |
+| `--scorecard-api-base`      | Base URL for scorecard API (API mode only)        | `https://api.securityscorecards.dev` |
+| `--scorecard-domain-prefix` | Domain prefix for repository URLs (API mode only) | `github.com`                         |
+| `--scorecard-http-timeout`  | HTTP timeout for API requests (API mode only)     | `30s`                                |
+
+### Global Flags
+
+| Flag                   | Description                                                                      | Default                                 |
+| ---------------------- | -------------------------------------------------------------------------------- | --------------------------------------- |
+| `--gql-addr string`    | Endpoint used to connect to GraphQL server                                       | `http://localhost:8080/query`           |
+| `--pubsub-addr string` | Address to connect to NATS pubsub service                                        | `nats://localhost:4222`                 |
+| `--blob-addr string`   | Address for blob storage                                                         | `file:///tmp/blobstore?no_tmp_dir=true` |
+| `--header-file string` | A text file containing HTTP headers to send to the GQL server, in RFC 822 format | Not set                                 |
+| `--publish-to-queue`   | Enable publishing to the message queue                                           | true                                    |
+
+## Usage Examples
```

Comment:
I'd leave the `--gql-addr` and `--pubsub-addr` flags out of your examples. These are the default values, so including them adds unnecessary complexity for the reader.